src/meta/src/manager/catalog/database.rs

```diff
@@ -101,6 +104,7 @@ impl DatabaseManager {
         let functions = Function::list(env.meta_store().as_kv()).await?;
         let connections = Connection::list(env.meta_store().as_kv()).await?;
         let subscriptions = Subscription::list(env.meta_store().as_kv()).await?;
+        let secrets = Secret::list(env.meta_store().as_kv()).await?;
```

Comment:
The refer count info of secret is not initialized. I found the implementation for connection is also wrong, you can leave a `TODO` here and I will rise a PR to fix connection first.

Reply:
Will be fixed after #16928 .

Reply:
Let me open a new pr to address this issue.

Reply:
will be resolved in https://github.com/risingwavelabs/risingwave/pull/16939 and merge into this pr